tasks/test/index.ts

```diff
@@ -25,9 +25,10 @@ async function main(): Promise<void> {
     let updateGoldens = task.getBoolInput('updateGoldens', false);
     let concurrency = task.getInput('concurrency', false);
     let canPublishTests = task.getInput('publishTests', false);
+    let canGenerateCodeCoverage = task.getInput("generateCodeCoverageReport", false);
```

Comment:
@chadrickman  task.getBoolInput instead of getInput as your canGenerate is of type boolean in task.json. Just a future reference :)